packages/koa-shopify-auth/src/auth/create-oauth-callback.ts

```diff
@@ -53,11 +53,17 @@ export default function createOAuthCallback(config: AuthConfig) {
     }
 
     const accessTokenData = await accessTokenResponse.json();
-    const {access_token: accessToken} = accessTokenData;
-
+    const {
+      access_token: accessToken,
+      associated_user_scope: associatedUserScope,
+      associated_user: associatedUser
+    } = accessTokenData;
+    
```

Comment:
```suggestion
    const {
      access_token: accessToken,
      associated_user_scope: associatedUserScope,
      associated_user: associatedUser,
    } = accessTokenData;

```

Yup it still says here that these things need to change, a comma on line 59 and extra spaces on line 61.